playlist/views.py

```diff
@@ -179,3 +181,24 @@ def post(self, *args, **kwargs):
         keyword = self.request.POST['keyword']
         playlists = Playlist.objects.filter(title__icontains=keyword)
         return render(self.request, self.template_name, {'playlists':playlists})
+
+
+class AddToPlaylistFromYoutube(LoginRequiredMixin, TemplateView):
+    """ Add to playlist from youtube search
+    """
+    template_name = 'playlist/search.html'
+
+    def post(self, *args, **kwargs):
+        playlist = get_object_or_404(
+            Playlist,
+            id=self.request.POST.get('playlist')
+        )
+        form = SongForm(
+            self.request.POST,
+            user=self.request.user,
+            playlist=playlist,
+        )
+        if form.is_valid():
+            form.save()
+            return JsonResponse({'result':'Successfully added song to playlist.'}, safe=False)
```

Comment:
move success message directly to template.

Reply:
Updated.